src/main/java/com/example/bookstore/dto/category/CategoryResponseDto.java

```diff
@@ -0,0 +1,10 @@
+package com.example.bookstore.dto.category;
+
+import lombok.Data;
+
+@Data
+public class CategoryResponseDto {
```

Comment:
I believe that this class is the same as `CategoryDto`, and I don't get the point of this